helpers/integration.go

```diff
@@ -71,6 +71,13 @@ type AppStudioTestResult struct {
 	Warnings  int    `json:"warnings"`
 }
 
+// IntegrationTestTaskResult provides results from integration test task
+// including metadata about validity of results
+type IntegrationTestTaskResult struct {
+	TestOutput      *AppStudioTestResult
+	ValidationError error
```

Comment:
Another way how to handle this issue, would be to return specific type of error from `GetTestResult()` and handle that error in upper levels, but I found it little bit harder to handle than just keeping validation error in struct.